<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageViewer</class>
 <widget class="QMainWindow" name="ImageViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>471</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>ImageViewer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>761</width>
         <height>376</height>
        </rect>
       </property>
       <widget class="QLabel" name="label_image">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>701</width>
          <height>331</height>
         </rect>
        </property>
        <property name="acceptDrops">
         <bool>true</bool>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>781</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="openAct"/>
    <addaction name="saveAct"/>
    <addaction name="saveasAct"/>
    <addaction name="printAct"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="quitAct"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="zoomInAct"/>
    <addaction name="zoomOutAct"/>
    <addaction name="separator"/>
    <addaction name="fitToWindowAct"/>
    <addaction name="normalSizeAct"/>
    <addaction name="rotClockwiseAct"/>
    <addaction name="rotAntiClockwiseAct"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_Software"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="openAct"/>
   <addaction name="saveAct"/>
   <addaction name="separator"/>
   <addaction name="zoomInAct"/>
   <addaction name="zoomOutAct"/>
   <addaction name="rotClockwiseAct"/>
   <addaction name="rotAntiClockwiseAct"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="openAct">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/icons/open.png</normaloff>:/images/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="saveAct">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/icons/save.png</normaloff>:/images/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="saveasAct">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/icons/Saveas.png</normaloff>:/images/icons/Saveas.png</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="printAct">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/icons/print.png</normaloff>:/images/icons/print.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="quitAct">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/icons/quit.png</normaloff>:/images/icons/quit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="zoomInAct">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/icons/zoom_in.png</normaloff>:/images/icons/zoom_in.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="toolTip">
    <string>Zoom in (25%)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="fitToWindowAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Fit to Window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="normalSizeAct">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Normal Size</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="rotClockwiseAct">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/icons/rot_clk.png</normaloff>:/images/icons/rot_clk.png</iconset>
   </property>
   <property name="text">
    <string>Rotate 90 clockwise</string>
   </property>
  </action>
  <action name="rotAntiClockwiseAct">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/icons/rot_anti.png</normaloff>:/images/icons/rot_anti.png</iconset>
   </property>
   <property name="text">
    <string>Rotate 90 Anticlockwise</string>
   </property>
  </action>
  <action name="actionAbout_Software">
   <property name="text">
    <string>About Software</string>
   </property>
  </action>
  <action name="zoomOutAct">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/icons/zoom_out.png</normaloff>:/images/icons/zoom_out.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
